.github/workflows/build-petclinic-overhead-base.yml

```diff
@@ -0,0 +1,30 @@
+name: publish-petclinic-base-image
+on:
+  push:
+    paths:
+      - 'testing-overhead/Dockerfile-petclinic-base'
+    branches: [ 'main' ]
+  workflow_dispatch:
+jobs:
+  push_to_registry:
+    name: make spring-petclinic-rest base image for overhead testing
+    runs-on: ubuntu-latest
+    permissions:
+      packages: write
+      contents: read
+    steps:
+      - name: check out the repo
+        uses: actions/checkout@v2
+      - uses: docker/setup-buildx-action@v1
+      - name: Login to GitHub Container Registry
+        uses: docker/login-action@v1
+        with:
+          registry: ghcr.io
+          username: ${{ github.repository_owner }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+      - name: push to gh packages
+        uses: docker/build-push-action@v2
+        with:
+          push: true
+          file: testing-overhead/Dockerfile-petclinic-base
+          tags: ghcr.io/open-telemetry/opentelemetry-java-instrumentation/petclinic-rest-base:latest
```

Comment:
I might be missing a new feature in GitHub which would be cool! But if not, we only have one image we use, `java-test-containers` and just use a tag per different image

https://github.com/open-telemetry/opentelemetry-java/blob/main/.github/workflows/docker-test-containers.yml#L33

ghcr is scoped to organization, or at least it was, so rather than asking for a new image each time we just do this.

Reply:
Yeah, it's scoped to org, but I _think_ this makes a package called `opentelemetry-java-instrumentation/petclinic-rest-base` - like the `/` is part of the name?  I can change it, but calling it `testcontainers` when it's really just petclinic seems misleading. 

How would you feel about trying the original approach and if it doesn't work I'll fix it?

Reply:
The problem is that we cannot automatically create new packages in otel org. Only org admins can do that.

Reply:
But we can try :)